ui/src/app/cohort-review/detail-tabs/detail-tabs.component.ts

```diff
@@ -315,7 +314,8 @@ export class DetailTabsComponent implements OnChanges, OnInit, OnDestroy {
     }
     this.subscription = this.route.data.map(({participant}) => participant)
       .subscribe(participants => {
-        this.participantsId = participants.participantId;
+        this.clickedParticipantId ? this.participantsId = this.clickedParticipantId
+          : this.participantsId = participants.participantId;
```

Comment:
nit: We can make this simpler
`this.participantsId = this.clickedParticipantId || participants.participantId;`

Reply:
ok let me check it.

Reply:
Done.